tools/src/tools.ml

```diff
@@ -44,7 +44,14 @@ type docItem =
           (** Additional documentation for constructors and record fields, if available. *)
     }
   | Module of docsForModule
-  | ModuleType of docsForModule
+  | ModuleType of {
+      id: string;
```

Comment:
Did you update the rescript types accordingly?

Reply:
https://github.com/rescript-lang/rescript-vscode/commit/a67efa57452f825da45bb7d8fd39a27895b17290